AdyenCard/Components/GiftCardComponent/GiftCardComponent.swift

```diff
@@ -312,4 +372,10 @@ public extension Result {
 }
 
 /// :nodoc:
-extension GiftCardComponent: TrackableComponent {}
+extension GiftCardComponent: TrackableComponent {
+    /// :nodoc:
+    public func viewDidLoad(viewController: UIViewController) {
+        Analytics.sendEvent(component: paymentMethod.type, flavor: _isDropIn ? .dropin : .components, environment: environment)
```

Comment:
you could actually call the default implementation of `TrackableComponent.viewDidLoad` here (kind of like a `super` method), since the signature of `viewDidLoad` is not in the protocol definition. but in #513 it is added there, so this won't work anymore :)

Reply:
In #513 is was actually wrong...

Reply:
yes, i see that you have now removed that :)